### ISSUE TITLE
✅ 🐞 Bug Report: Testimonial Widget's Style tab has a white space on the top - [ED-11956]

### ISSUE THREAD

**Issue by iqbal-web** [elementor/elementor#23402]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

here a screenshort https://prnt.sc/OcEdOD26OZAO
looks like there hide a section & more white space, same problem many widget

### Steps to reproduce

Got to elementor testimonial widget then go style section,

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info
<details>
<summary>Click to reveal</summary>

````txt

### wp-core ###

version: 6.3
site_language: en_US
user_language: en_US
timezone: +00:00
permalink: /%postname%/
https_status: false
multisite: false
user_registration: 0
blog_public: 1
default_comment_status: open
environment_type: production
user_count: 1
dotorg_communication: true

### wp-paths-sizes ###

wordpress_path: F:\xampp\htdocs\xs-test
wordpress_size: 52.04 MB (54568229 bytes)
uploads_path: F:\xampp\htdocs\xs-test/wp-content/uploads
uploads_size: 93.40 KB (95646 bytes)
themes_path: F:\xampp\htdocs\xs-test/wp-content/themes
themes_size: 12.35 MB (12951623 bytes)
plugins_path: F:\xampp\htdocs\xs-test/wp-content/plugins
plugins_size: 67.76 MB (71050124 bytes)
database_size: 3.83 MB (4014080 bytes)
total_size: 136.07 MB (142679702 bytes)

### wp-active-theme ###

name: Twenty Twenty-Three (twentytwentythree)
version: 1.1 (latest version: 1.2)
author: the WordPress team
author_website: https://wordpress.org
parent_theme: none
theme_features: core-block-patterns, post-thumbnails, responsive-embeds, editor-styles, html5, automatic-feed-links, block-templates, widgets-block-editor
theme_path: F:\xampp\htdocs\xs-test/wp-content/themes/twentytwentythree
auto_update: Disabled

### wp-themes-inactive (2) ###

Twenty Twenty-One: version: 1.8, author: the WordPress team, Auto-updates disabled
Twenty Twenty-Two: version: 1.4, author: the WordPress team, Auto-updates disabled

### wp-plugins-active (3) ###

Elementor: version: 3.15.1, author: Elementor.com (latest version: 3.15.2), Auto-updates disabled
ElementsKit Lite: version: 2.9.0, author: Wpmet, Auto-updates disabled

### wp-media ###

image_editor: false
imagick_module_version: Not available
imagemagick_version: Not available
imagick_version: Not available
file_uploads: File uploads is turned off
post_max_size: 40M
upload_max_filesize: 40M
max_effective_size: 40 MB
max_file_uploads: 20
gd_version: not available
ghostscript_version: not available

### wp-server ###

server_architecture: Windows NT 10.0 AMD64
httpd_software: Apache/2.4.56 (Win64) OpenSSL/1.1.1t PHP/8.1.17
php_version: 8.1.17 64bit
php_sapi: apache2handler
max_input_variables: 1000
time_limit: 120
memory_limit: 512M
max_input_time: 60
upload_max_filesize: 40M
php_post_max_size: 40M
curl_version: 7.77.0 OpenSSL/1.1.1t
suhosin: false
imagick_availability: false
pretty_permalinks: true
htaccess_extra_rules: false
current: 2023-08-13T18:11:33+00:00
utc-time: Sunday, 13-Aug-23 18:11:33 UTC
server-time: 2023-08-13T18:11:31+00:00

### wp-database ###

extension: mysqli
server_version: 10.4.28-MariaDB
client_version: mysqlnd 8.1.17
max_allowed_packet: 1048576
max_connections: 151

### wp-constants ###

WP_HOME: undefined
WP_SITEURL: undefined
WP_CONTENT_DIR: F:\xampp\htdocs\xs-test/wp-content
WP_PLUGIN_DIR: F:\xampp\htdocs\xs-test/wp-content/plugins
WP_MEMORY_LIMIT: 40M
WP_MAX_MEMORY_LIMIT: 512M
WP_DEBUG: false
WP_DEBUG_DISPLAY: true
WP_DEBUG_LOG: false
SCRIPT_DEBUG: false
WP_CACHE: false
CONCATENATE_SCRIPTS: undefined
COMPRESS_SCRIPTS: undefined
COMPRESS_CSS: undefined
WP_ENVIRONMENT_TYPE: Undefined
WP_DEVELOPMENT_MODE: undefined
DB_CHARSET: utf8mb4
DB_COLLATE: undefined

### wp-filesystem ###

wordpress: writable
wp-content: writable
uploads: writable
plugins: writable
themes: writable


````

</details>

**Comment by nicholaszein** [comment]
Hello @iqbal-web,

Thank you for reaching out! 🙏 

### ⛔ While reviewing your report we concluded this is not an issue caused by Elementor. Here's what we found out:
- That's the expected behavior. All widgets have a space above the first section to give separation between the widget tabs and the content inside each one.

### 👍 For now, that's all we can help with. 

Kind regards

**Comment by nicholaszein** [comment]
Hello there @iqbal-web!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor v3.17! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
